usaspending_api/api_contracts/contracts/v2/references/filter_tree/psc/group.md

```diff
@@ -0,0 +1,73 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# PSC
+
+This endpoint is used to power USAspending.gov's "Product or Service Code" search component on the advanced search page.
+The response is a forest of search filter nodes, which despite having a unified structure represent different
+database fields based on depth in the tree.
+
+## Toptier Search [GET /api/v2/references/filter_tree/psc/{group}/{?depth}{?filter}]
+
+Returns PSCs under the provided group
++ Request (application/json)
+    + Parameters
+        + `group`: `Product` (required, string)
+        Parent group of PSCs to return
+        + `depth` (optional, number)
+            Defines how many levels of descendants to return under each node. For example, depth=0 will 
+            return a flat array, while depth=2 will populate the children array of each top level node 
+            with that node's children and grandchildren. The actual depth of each tree may be less than 
+            the value of depth if returned nodes have no children. Negative values are treated as 
+            infinite, returning all descendants.  
+            + Default: 0
+        
+        + `filter` (optional, string) 
+            Restricts results to nodes with a `id` or `description` matching the filter string. If depth is 
```

Comment:
What is the `id` field, is it the PSC?

Reply:
Unless it's the group name. Like in the TAS filter tree, the meaning of `id` and `description` are context-dependent, hence their more generic naming.